packages/core/content-type-builder/admin/src/components/FormModalSubHeader/index.js

```diff
@@ -1,11 +1,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { useIntl } from 'react-intl';
-import { Typography } from '@strapi/design-system/Typography';
+import styled from 'styled-components';
 import upperFirst from 'lodash/upperFirst';
+import { Flex } from '@strapi/design-system/Flex';
+import { Typography } from '@strapi/design-system/Typography';
+
 import { getTrad } from '../../utils';
+
 import getModalTitleSubHeader from './getModalTitleSubHeader';
 
+const SubHeaderContainer = styled(Flex)`
+  margin-bottom: ${(props) => props.theme.spaces[2]};
```

Comment:
mmh I think we could use `paddingBottom` prop, and to handle the space between `Subtitle` and the other `Typography` we could base `SubHeaderContainer` on `Stack` component 👀

```js
<Stack direction="column" alignItems="flex-start" paddingBottom={2} spacing={1}>
  <Typography variant="beta" />
  <Typography variant="pi" />
</Stack>
```